packages/integration-tests/test/rpc.spec.ts

```diff
@@ -161,24 +160,27 @@ describe('Transactions', () => {
 
   // This test depends on previous transactions being mined
   it('should get block with transactions', async () => {
-      const block = await provider.getBlockWithTransactions('latest')
-      assert(block.number !== 0)
-      // ethers JsonRpcProvider does not return the state root
-      // but the OptimismProvider does.
-      assert(typeof block.stateRoot === 'string')
-      // There must be a single transaction
-      assert(block.transactions.length === 1)
-      const tx = block.transactions[0]
-      // The previous test sends a transaction directly to L2 so
-      // the l1BlockNumber is null
-      assert(tx.l1BlockNumber === null)
-      // The `OptimismProvider` creates EthSign transactions
-      assert(tx.txType === 'EthSign')
-      // The transaction was sent directly to the sequencer so the
-      // queue origin is sequencer
-      assert(tx.queueOrigin === 'sequencer')
-      // The queue origin is the sequencer, not L1, so there is
-      // no L1TxOrigin
-      assert(tx.l1TxOrigin === null)
+    const block = await provider.getBlockWithTransactions('latest')
+    assert(block.number !== 0)
+    // ethers JsonRpcProvider does not return the state root
+    // but the OptimismProvider does.
+    assert(typeof block.stateRoot === 'string')
+    // There must be a single transaction
+    assert(block.transactions.length === 1)
+    const tx = block.transactions[0]
+
+    // TODO - This assert is broken
```

Comment:
Be sure to rebase on master because this was removed here: https://github.com/ethereum-optimism/integration-tests/commit/b4e16141b750fc9d491b81540212ebb2b91a1d5d

Reply:
aha nice. what's the team's preferred workflow for rebasing?